ibis_substrait/compiler/translate.py

```diff
@@ -583,6 +583,36 @@ def unbound_table(
     )
 
 
+@translate.register(ops.LocalTable)
+def local_table(
+    op: ops.LocalTable, expr: ir.TableExpr, _: SubstraitCompiler, **kwargs: Any
+) -> stalg.Rel:
+    def path_type_of(name : str | None):
```

Comment:
These functions should be defined outside of the `local_table` function with `_` prefixed to the name (`_path_type_of` instead of `path_type_of`) to indicate that they are not public APIs.